src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -114,42 +109,21 @@ export class Kernel implements IKernel {
         const notebookPromise = this.startNotebook({ disableUI: false });
         await this.kernelExecution.executeAllCells(notebookPromise, document);
     }
-    public async cancelCell(cell: NotebookCell) {
-        this.startCancellation.cancel();
-        await this.kernelExecution.cancelCell(cell);
-    }
-    public async cancelAllCells(document: NotebookDocument) {
-        this.startCancellation.cancel();
-        await this.kernelExecution.cancelAllCells(document);
-    }
     public async start(options?: { disableUI?: boolean }): Promise<void> {
         await this.startNotebook(options);
     }
-    public async interruptCell(cell: NotebookCell): Promise<InterruptResult> {
```

Comment:
Removed cancelCell, cancelAllCells, interruptCell & interruptAllCells
In Jupyter, its all the same - `interrupt`.
Hence collapsed all into a single method